### DISCUSSION TITLE
NEXT css is loading after the page

### DISCUSSION THREAD

**Discussion by LaurineObriot** [vercel/next.js#58738] [Help]
### Summary

I use Next version 14.0.3, with react version 18.2.0. I'm trying to create a very classic static site. I have a header that allows me to go to all my pages. 

Problem, when I click on a link to change page, I'm redirected, and then for 1 or 2 seconds I see the page in html, without the css. The css is loaded afterwards. So I'd like the page to be visible only when the css is loaded. 
The strange thing is that it works fine locally, but not on the deployed version.

I use material ui and scss modules. 

I've tried using dynamic component loading, but it doesn't make any difference (import dynamic from 'next/dynamic').
<img width="1430" alt="Capture d’écran 2023-11-21 à 17 49 18" src="https://github.com/vercel/next.js/assets/53184615/86bbfc48-4560-4399-9031-e5b0b52ac816">

**Comment by mutasim77** [comment]
check it out: [issue](https://github.com/mui/material-ui/issues/30410#issuecomment-1148489813)
